### PR TITLE
Add a test suite

### PR DESCRIPTION
**Pull request by blink1073**
100% coverage.

### CODE REVIEW

**Comment by blink1073** [comment]
Hold on, I don't need the css for the test.

**Comment by blink1073** [comment]
Ready.

File: test/src/index.ts [hunk 1]
```diff
@@ -9,12 +9,574 @@
 
 import expect = require('expect.js');
 
+import {
+  Message, postMessage, sendMessage
```
Comment (sccolbert) [inline test/src/index.ts]: postMessage is unused

**Comment by blink1073** [comment]
Fixed.

**Pull request merged**